test/t8n/t8n.cpp

```diff
@@ -128,6 +136,16 @@ int main(int argc, const char* argv[])
                 }
             }
 
+            if (state_reward != 0 && state_reward != std::numeric_limits<intx::uint256>::max())
+                state.get_or_insert(block.coinbase).balance += state_reward;
+            else if (state_reward == 0)
+                state.touch(block.coinbase);
+            else if (state_reward == std::numeric_limits<intx::uint256>::max())
+            {
+                if (state.get_or_insert(block.coinbase).is_empty())
```

Comment:
Hm, cleaning up empty accounts is just an EIP-161 rule. Don't we have that implemented?

Reply:
For `<= EIP158` coinbase account is touched even it tx is rejected. retesteth passing -1 as a stare.reward requires zero state to be removed.